plugin.go

```diff
@@ -6,7 +6,6 @@ import (
 	"github.com/aymerick/douceur/inliner"
 	"github.com/drone/drone-go/template"
 	"github.com/jaytaylor/html2text"
-	"gopkg.in/gomail.v2"
```

Comment:
I just tested your code, and the removal of this import is giving me issues. Did you do this for a reason, or did it just slip through by accident?

Reply:
Ah, Goland's optimize imports got me somehow! I just readded this line.